package.json

```diff
@@ -113,6 +113,34 @@
             {
                 "command": "clangd.activate",
                 "title": "Manually activate clangd extension"
+            },
+            {
+                "command": "clangd.typeHierarchy",
+                "title": "Open Type Hierarchy"
+            },
+            {
+                "command": "clangd.typeHierarchy.viewParents",
+                "title": "Show Base Classes",
+                "icon": {
+                  "light": "resources/base-light.svg",
+                  "dark": "resources/base-dark.svg"
+                }
+            },
+            {
+                "command": "clangd.typeHierarchy.viewChildren",
+                "title": "Show Derived Classes",
+                "icon": {
+                  "light": "resources/derived-light.svg",
+                  "dark": "resources/derived-dark.svg"
+                }
+            },
+            {
+                "command": "clangd.typeHierarchy.close",
+                "title": "Close",
+                "icon": {
+                  "light": "resources/close-light.svg",
+                  "dark": "resources/close-dark.svg"
+                }
             }
         ],
         "keybindings": [
```

Comment:
You may want to bind a key for this, call hierarchy is shift-alt-h. Is shift-alt-t free?